test/import-map.test.js

```diff
@@ -4,22 +4,24 @@ const {
   resetManifest: resetMemoryManifest,
 } = require("../src/io-methods/memory");
 
-beforeAll(() => {
-  setConfig({
-    manifestFormat: "importmap",
-    packagesViaTrailingSlashes: true,
-    locations: {
-      prod: "memory://prod",
-    },
+describe(`/import-map.json`, () => {
+  let errorSpy;
```

Comment:
Added this errorSpy so the tests wouldn't actually error to the console and clutter up the output

Reply:
I wanted to scope it as well so I moved the `beforeAll` and `beforeEach` blocks into the `describe` block